apis/python/tests/_util.py

```diff
@@ -1,12 +1,43 @@
 from contextlib import contextmanager, nullcontext
 from pathlib import Path
-from typing import Any, Tuple, Type, Union
+from typing import Any, List, Optional, Tuple, Type, Union
 
+import pandas as pd
 import pytest
 from _pytest._code import ExceptionInfo
 from _pytest.python_api import E, RaisesContext
 from typeguard import suppress_type_checks
 
+
+def parse_col(col_str: str) -> Tuple[Optional[str], List[str]]:
+    """Parse a "column string" of the form `val1,val2,...` or `name=val1,val2,...`."""
+    pcs = col_str.split("=")
+    if len(pcs) == 1:
+        return None, col_str.split(",")
+    elif len(pcs) == 2:
+        name, vals_str = pcs
+        vals = vals_str.split(",")
+        return name, vals
+    else:
+        raise ValueError(f"Invalid column string: {col_str}")
+
+
+def make_df(index_str: Optional[str] = None, **cols) -> pd.DataFrame:
```

Comment:
```suggestion
def make_pandas_dataframe(index_str: Optional[str] = None, **cols) -> pd.DataFrame:
```

Reply:
I don't like the short name but you're right it's the name as before, just moved from one file to another.

"df" is ambigious: there are Pandas and SOMA "dfs". I'll put up a separate PR.